db/migrate/20251002140125_add_hide_from_search_index_field_to_documents.rb

```diff
@@ -0,0 +1,5 @@
+class AddHideFromSearchIndexFieldToDocuments < ActiveRecord::Migration[8.0]
+  def change
+    add_column :documents, :hide_from_search_index, :boolean, default: false, null: false
```

Comment:
is it most straightforward to think of this flag in terms of `hide_from_search_index` or as something like `testing_artefact`? Perhaps that depends whether / how we are planning to ultimately delete these content blocks?